File: Languages/C++/C++Abstractions/Ch1/exercises/ex1.cpp
/*
Write a program that reads in a temperature in degrees Celsius and displays the
corresponding temperature in degrees Fahrenheit. The conversion formula is
F = C + 32
*/

#include <iostream>
#include <string>
using std::string;
//using std::string_literals;
using std::cout;
using std::cin;
using std::endl;

int main()
{
    int celsius = 0, fahrenheit = 0;

    //user input
    cout << "Enter your Celsius to convert to Fahrenheit: ";
    while (cin >> celsius)
    {
        cout << celsius << " Celsius = " << celsius + 32 << " Fahrenheits\n\n";
        
        cout << "Enter your Celsius to convert to Fahrenheit [ctrl + D to quit]: ";
    }

}

File: Languages/python/zelle/ch2/ex1.py
# convert.py
# A program to convert Celsius temps to Fahrenheit

""" 1. A user-friendly program should print an introduction that tells the user
what the program does. Modify the convert . py program (Section 2.2) to
print an introduction.  """


def main():
    print("This program converts celcius temperatures to Fahrenheit")
    celsius = eval(input("What is the Celsius temperature? "))
    fahrenheit = 9/5 * celsius + 32
    print("The temperature is ", fahrenheit, "degrees fahrenheit.")

main()